scylla-cdc-base/src/test/java/com/scylladb/cdc/model/worker/MockRawChange.java

```diff
@@ -109,6 +111,22 @@ public Object getAsObject() {
             };
         }
 
+        private Set<Field> boxObjectsToFields(Set<Object> set, ChangeSchema.DataType setDataType) {
+            if (set == null) {
+                return null;
+            }
+            // Using LinkedHashSet to preserve the order.
```

Comment:
Does the order matter in those sets? In general when Set is used the order shouldn't matter and when it does it's just a bug.

Reply:
In this case (`MockRawChange`) I used this to make writing tests easier (you don't have to guess the order).